Twig/LocalAssetTwigExtension.php

```diff
@@ -54,26 +54,33 @@ public function getFunctions()
     public function getFilters()
     {
         return [
-            new \Twig_SimpleFilter('abbr_class', [$this, 'getLocalAsset']),
```

Comment:
whats that? shouldnt we add a line to the upgrade that this was removed?

Reply:
That is basically a copy & paste fail. `abbr_class` is a [default twig filter](https://symfony.com/doc/current/reference/twig_reference.html#abbr-class) and so should still exist.